provider/resources.go

```diff
@@ -1235,7 +1239,7 @@ compatibility shim in favor of the new "name" field.`)
 						AltTypes: []tokens.Type{awsResource(ec2Mod, "PlacementGroup")},
 					},
 					"enabled_metrics": {
-						Elem: &tfbridge.SchemaInfo{Type: awsType(autoscalingMod, "metrics", "Metric")},
+						Elem: &tfbridge.SchemaInfo{Type: awsType(autoscalingMod, "Metric", "Metric")},
```

Comment:
A dangling reference is replaced with an actual reference to a real enum here, as one possibility to fix the problem. The other possibility would be to introduce a IsOverlay entry for the previous dangling reference.